docs/src/content/docs/commands/KEYS.md

```diff
@@ -13,57 +13,65 @@ KEYS pattern
 
 ## Parameters
 
-- `pattern`: A string that represents the pattern to match against the keys in the DiceDB database. The pattern can include special glob-style characters:
-  - `*` matches any number of characters (including zero).
-  - `?` matches exactly one character.
-  - `[abc]` matches any one of the characters inside the brackets.
-  - `[a-z]` matches any character in the specified range.
+| Parameter       | Description                                      | Type    | Required |
+|-----------------|--------------------------------------------------|---------|----------|
+| `pattern`           | A string representing the pattern to match against the keys in DiceDB databse.| String  | Yes      |
 
-## Return Value
+Supported glob-style characters:
```

Comment:
And here also, we are explicitly mentioning this.

Reply:
Hey, thanks for pointing this one out. I did not see this. I would suggest adding a description as below:
`Use \ to escape special characters if you want to match them verbatim.`